src/Tags/Vite.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Statamic\Tags;
+
+use Illuminate\Foundation\Vite as LaravelVite;
+
+class Vite extends Tags
+{
+    /**
+     * The {{ vite }} tag.
+     *
+     * @return string|array
+     */
+    public function index()
+    {
+        if (! $src = $this->params->get('src')) {
```

Comment:
It's my understanding that you may want to provide multiple sources.

For example on the Blade directive you could do this:

```blade
@vite(['resources/css/app.css', 'resources/js/app.js'])
```

So, you'll probably want to change this line to use `$this->params->explode('src')` so you can do `src="app.css|app.js"`